src/common/models/pro_meta.go

```diff
@@ -36,5 +36,4 @@ type ProjectMetadata struct {
 	Value        string    `orm:"column(value)" json:"value"`
 	CreationTime time.Time `orm:"column(creation_time);auto_now_add" json:"creation_time"`
```

Comment:
move the model to pkg as well?

Reply:
The functions of the project in `common/dao` still use these models so these models will not be moved to pkg until the functions in `common/dao` were removed.